.github/workflows/docker.yml

```diff
@@ -87,4 +90,13 @@ jobs:
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
           build-args: |
-            VERSION=latest
\ No newline at end of file
+            VERSION=latest
+
+      - name: Build OpenTelemetry Collector with APIClarity plugin
+        uses: docker/build-push-action@v3
+        with:
+          context: plugins
+          tags: otel-apiclarityexporter:latest,otel-apiclarityexporter:$GITHUB_SHA,otel-apiclarityexporter:$OTEL_COLLECTOR_VERSION
```

Comment:
you are missing the `file` (docker file path), `cache-from` and `cache-to`. you can look at `Build passive taper`

Reply:
In [76ce43d](https://github.com/openclarity/apiclarity/pull/255/commits/76ce43d1be9f34c99a5d51006373dc5414768c1c)